content/docs/dvclive/user-guide/dvclive-with-dvc.md

```diff
@@ -0,0 +1,129 @@
+- DVCLive is also capable of generating [_checkpoint_](#checkpoints) signal
+  files used by DVC <abbr>experiments<abbr>.
+
+## Setup
+
+We will refer to a training script (`train.py`) already using `dvclive`:
+
+```python
+# train.py
+
+import dvclive
+
+for epoch in range(NUM_EPOCHS):
+    train_model(...)
+    metrics = evaluate_model(...)
+
+    for metric_name, value in metrics.items():
+        dvclive.log(metric_name, value)
+
+    dvclive.next_step()
+```
+
+> 💡 When using DVCLive in a DVC project, there is no need for manual
+> initialization of DVCLive (no [`dvclive.init()`] call). DVCLive will use the
+> settings provided by DVC.
+
+Let's use `dvc stage add` to create a stage to wrap this code (don't forget to
+`dvc init` first):
+
+```dvc
+$ dvc stage add -n train --live training_metrics
+                -d train.py python train.py
+```
+
+`dvc.yaml` will contain a new `train` stage with the [`DVCLive configuration`]
+(in the `live` field):
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    live:
+      training_metrics:
+        summary: true
+        html: true
+```
+
+The value passed to `--live` (`training_metrics`) became the directory `path`
+for DVCLive to write logs in, and DVC will now
+[track](/doc/use-cases/versioning-data-and-model-files) it. Other supported
+command options for the DVC integration:
+
+- `--live-no-cache <path>` - specify a DVCLive log directory `path` but don't
+  tracked it with DVC. Useful if you prefer to track it with Git.
+- `--live-no-summary` - passes `summary=False` to [`DVCLive configuration`].
+- `--live-no-html` - passes `html=False` to [`DVCLive configuration`].
+
+> Note that these are convenience CLI options. You can still use
+> [`dvclive.init()`] manually, which will override any options sent to
+> `dvc stage add`. Just be careful to match the `--live` value (CLI) and `path`
+> argument (code). Also, note that summary files are never tracked by DVC
+> automatically.
+
+Run the training with `dvc repro`:
+
+```bash
+$ dvc repro train
+```
+
+## Outputs
+
+After that's finished, you should see the following content in the project:
+
+```bash
+$ ls
+dvc.lock  training_metrics       training_metrics.json
+dvc.yaml  training_metrics.html  train.py
+```
+
+The `.tsv` files generated under `training_metrics` can be visualized with
+`dvc plots`.
+
+In addition, `training_metrics.json` can be used by `dvc metrics`.
+
+### DVC Studio
+
+[DVC Studio](/docs/studio) will automatically parse the outputs generated by
+DVCLive, allowing to
+[compare and visualize](/docs/studio/user-guide/visualize-experiments)
+experiments using DVCLive in DVC Studio.
+
+### HTML report
+
+In addition to the
+[outputs described in the Quickstart](/docs/dvclive/user-guide/quickstart#outputs),
+DVC generates an _HTML report_.
+
+If you open `training_metrics.html` in a browser, you'll see a plot for metrics
+being logged during the model training!
+
+![](/img/dvclive_report.png)
```

Comment:
Some possible improvements here (could be in a subsequent PR):

* Two plots is probably enough.
* More steps would probably look better.
* Can we do a gif or something that shows the plot over time?

Reply:
A gif could be really cool. The html report however is not automatically refreshed, though. If you open it in a browser you need to manually refresh (although it might be easy to add a small javascript code to the html template in order to force the auto refreshing)

Reply:
Yeah, maybe a gif suggests automatic refresh. Should we mention that users should refresh to get an updated plot?

Reply:
Added a note for now. Opened P.R. on dvc for supporting auto refreshing https://github.com/iterative/dvc/pull/6400